src/main/java/org/apache/pirk/querier/wideskies/QuerierProps.java

```diff
@@ -0,0 +1,202 @@
+package org.apache.pirk.querier.wideskies;
```

Comment:
Hmm, need to investigate why RAT checks are not working.
I can take a look tomorrow.